tests/store/artifact/test_databricks_artifact_repo.py

```diff
@@ -1390,7 +1396,7 @@ def test_multipart_upload_abort(databricks_artifact_repo, large_file, mock_chunk
         f"{DATABRICKS_ARTIFACT_REPOSITORY}._call_endpoint",
         side_effect=[create_mpu_response, Exception("Failed to complete multipart upload")],
     ) as call_endpoint_mock, mock.patch(
-        "requests.Session.request", side_effect=mock_responses
+        "requests.Session.request", side_effect=mock_request
```

Comment:
In this test, we make two requests for part uploads. The existing code no longer works because the request for the second part upload comes in first before the first part upload due to parallelization.